### PR TITLE
Add inifile recipe

### PR DESCRIPTION
**Pull request by goanpeca**

### CODE REVIEW

File: recipes/inifile/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  run:
+    - python
+
+test:
+  imports:
+    - inifile
+
+about:
+  home: https://github.com/mitsuhiko/python-inifile
+  license: BSD 3-Clause
```
Comment (jakirkham) [inline recipes/inifile/meta.yaml]: Could we please ask upstream to include a `MANIFEST.in` that lists the license file so it is included in packages (e.g. `sdist`s)?

Comment (goanpeca) [inline recipes/inifile/meta.yaml]: Sure we could :-p

**Comment by goanpeca** [comment]
Merge @jakirkham ?

**Comment by jakirkham** [comment]
Please raise the issue mentioned above. Then will merge once Travis clears up a bit.

File: recipes/inifile/meta.yaml (hunk 1 above)
Comment (goanpeca) [inline recipes/inifile/meta.yaml]: https://github.com/mitsuhiko/python-inifile/issues/6

Comment (jakirkham) [inline recipes/inifile/meta.yaml]: Thanks.

**Comment by jakirkham** [comment]
Thanks @goanpeca.

**Pull request merged**